package.json

```diff
@@ -27,6 +27,7 @@
     "cross-spawn": "^4.0.0",
     "debug": "2.2.0",
     "delocalize-dependencies": "0.1.0",
+    "flatten": "1.0.2",
```

Comment:
i am suggesting to use more perfed (arr-flatten)[https://github.com/jonschlinkert/arr-flatten] by @jonschlinkert